elementary/config/config.py

```diff
@@ -91,3 +93,8 @@ def _validate_elementary_profile(self):
                 raise NoElementaryProfileError
         except FileNotFoundError:
             raise NoProfilesFileError(self.profiles_dir)
+
+    def validate_timezone(self):
+      if self.slack_timezone is not None:
```

Comment:
Maybe here we should use `dateutil.tz.gettz()` at it's actually used later on, and verify that it's not `None` as returned when a tz could not be found? 

```py
if not dateutil.tz.gettz(self.slack_timezone):
   raise InvalidArgumentsError('...')
```

Reply:
Your suggestion is much more elegant! 
I made the change thx!